src/components/storyCard/storyCard.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
instead of adding it here, let's do it in the eslint config file, please.

Reply:
Also, I think for Next.js, we don't require the React import.... 😄